package.json

```diff
@@ -6,7 +6,7 @@
   "types": "./dist/index.d.ts",
   "scripts": {
     "build": "tsc -d",
-    "build:watch": "yarn build -w",
```

Comment:
can we just make this `tsc -dw` instead so folks can use whatever CLI they want?